PubNub/Network/PNNetwork.m

```diff
@@ -362,6 +363,23 @@ - (Class)resultClassForOperation:(PNOperationType)operation;
  */
 - (Class)statusClassForOperation:(PNOperationType)operation;
 
+/// Process passed operation using set of parameters.
+///
+/// Translate client operation to actual request to \b PubNub network.
+///
+/// - Parameters:
+///   - operationType: One of `PNOperationType` enumerator fields which describe what kind of operation should be
+///     executed by client.
+///   - parameters: Request parameters representation object.
+///   - data: Binary data which should be pushed to **PubNub** network along with request.
+///   - block: Depending on operation type it can be `PNResultBlock`, `PNStatusBlock` or `PNCompletionBlock` blocks.
+/// - Since: 5.3.0
+- (void)processOperation:(PNOperationType)operationType
```

Comment:
It's a minor thing, but the documentation for `retryAttempt` parameter is missing. But it's obvious so you don't have to fix it